sql/2021/compression/compression_format_trend.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+  # compression_format_trend.sql : What compression formats are being used (gzip, brotli, etc)
+SELECT
+  EXTRACT(YEAR FROM date) AS year,
+  client,
+  IF(resp_content_encoding = "gzip", "Gzip", IF(resp_content_encoding = "br", "Brotli", IF(resp_content_encoding = "", "no text compression", "other"))) AS compression_type,
```

Comment:
Nit: this if/else chain lends itself well to CASE/WHEN/ELSE statements instead and is a bit easier to read
```suggestion
  CASE
    WHEN resp_content_encoding = 'gzip' THEN 'Gzip'
    WHEN resp_content_encoding = 'br' THEN 'Brotli'
    WHEN resp_content_encoding = '' THEN 'no text compression'
    ELSE 'other'
  END AS compression_type,
```